roxctl/maincommand/command.go

```diff
@@ -90,6 +91,9 @@ func Command() *cobra.Command {
 	if env.DeclarativeConfiguration.BooleanSetting() {
 		c.AddCommand(declarativeconfig.Command(cliEnvironment))
 	}
+	if !buildinfo.ReleaseBuild {
```

Comment:
why to hide this?

Reply:
So, for the time being this is only for the docs team to have some form of input, and I'm not sure whether it's too worthwhile to have for end-users. If we enable this for release builds we also have to maintain it, hence left it for non-release builds only now.